gammapy/background/models.py

```diff
@@ -650,3 +651,26 @@ def compute_rate(self):
         bg_rate = bg_rate.to('MeV-1 sr-1 s-1')
 
         self.bg_rate.data = bg_rate
+
+    def smooth(self):
+        """Smooth the bkg rate with a gaussian 1D kernel.
+
+        """
+        for binE, E in enumerate(self.counts.energy[:-1]):
+            counts = self.counts.data[binE, :]
+            Nev = np.sum(counts).value
+            if (Nev > 0):
```

Comment:
Is this a speed optimisation to skip if `Nev == 0`?
Or does the code error if you execute a bin with `Nev == 0`?
Maybe you can leave a one-line comment like this (if this is accurate, I'm not sure):
```
# For zero counts, the background rate is zero and smoothing would not change it.
# For speed we're skipping the smoothing in that case
```